relay-config/src/config.rs

```diff
@@ -545,11 +545,6 @@ struct Limits {
     /// The maximum number of seconds to wait for pending envelopes after receiving a shutdown
     /// signal.
     shutdown_timeout: u64,
-    /// The maximum memory allocation Relay can make, in bytes.
-    ///
-    /// Purposefully undocumented because we don't know yet how feasible this is.
-    #[serde(rename = "_max_alloc_size")]
-    max_alloc_size: Option<ByteSize>,
```

Comment:
If someone has this option in their config file, it will just be ignored from now on, right?

Reply:
Correct.